framework_tvb/tvb/interfaces/web/static/js/bursts.js

```diff
@@ -946,7 +946,17 @@ function calculateValuesInRage(pse_param_lo, pse_param_hi, pse_param_step){
     return pse_param_number;
 }
 
-function displayPseSimulationMessage() {
+function _getRangeValueForGuidParameter(pse_param_guid){
```

Comment:
I was thinking to get more in this function.
Including the parts where values are read from HTML $("#pse_param1_hi")[0].valueAsNumber;...

Reply:
So should I read those params dynamically as well? Like sending "pse_param1" as parameter and then compute the rest in a separate function from there?

Reply:
yes, something like `range_no = compute_range_no_for_param_prefix('pse_param1')`